include/GafferTest/TypeIds.h

```diff
@@ -45,7 +45,7 @@ enum TypeId
 
 	MultiplyNodeTypeId = 110201,
 
-	LastTypeId = 110250,
```

Comment:
Huh, are we really this tight on available ids?   We probably don't need 25 more test nodes, but I wasn't expecting to need to be stingy.

Reply:
Currently, Gaffer is limited to 1000 TypeIds, carved out in Cortex here : https://github.com/ImageEngine/cortex/blob/main/include/IECore/TypeIds.h#L287-L288. Obviously we can replace that range with a bigger one, but I'd rather not have to do that bit of juggling right now.